content/tsColinPrecip.md

```diff
@@ -27,12 +25,21 @@ Function to retrieve precip data using [`geoknife`](https://github.com/USGS-R/ge
 ``` r
 getPrecip <- function(states, startDate, endDate){
   
-  wg_s <- webgeom(geom = 'derivative:US_Counties', attribute = 'STATE')
-  wg_c <- webgeom(geom = 'derivative:US_Counties', attribute = 'COUNTY')
+  # fips webgeom
   wg_f <- webgeom(geom = 'derivative:US_Counties', attribute = 'FIPS')
-  county_info <- data.frame(state = query(wg_s, 'values'), county = query(wg_c, 'values'), 
-                            fips = query(wg_f, 'values'), stringsAsFactors = FALSE) %>% 
-    unique() 
+  
+  # use fips data from US Census Bureau (found in Data.gov catalog:
+  # https://catalog.data.gov/dataset/fips-county-code-look-up-tool)
+  fips_lookup <- data.frame(maps::county.fips, stringsAsFactors = FALSE) %>% 
```

Comment:
oh yeah, whoops. That's left from when I was trying to see if I could have it not bring `polynames` in as a factor.